src/homepageExperience/components/steps/CreateToken.tsx

```diff
@@ -0,0 +1,73 @@
+// Libraries
+import React, {FC} from 'react'
+import CodeSnippet from 'src/shared/components/CodeSnippet'
+import {Button, ComponentColor, ComponentSize} from '@influxdata/clockface'
+import {connect, ConnectedProps, useDispatch, useSelector} from 'react-redux'
+import {RouteComponentProps, withRouter} from 'react-router-dom'
+
+// Selectors
+import {notify} from 'src/shared/actions/notifications'
+import {getResourcesTokensFailure} from 'src/shared/copy/notifications'
+import {getAllResources} from 'src/authorizations/actions/thunks'
+import {dismissOverlay, showOverlay} from 'src/overlays/actions/overlays'
+import {getOrg} from 'src/organizations/selectors'
+
+type ReduxProps = ConnectedProps<typeof connector>
+
+const CreateToken: FC<ReduxProps & RouteComponentProps> = ({
+  showOverlay,
+  dismissOverlay,
+  getAllResources,
+}) => {
+  const org = useSelector(getOrg)
+  const dispatch = useDispatch()
+  const handleAllAccess = async () => {
+    try {
+      await getAllResources()
+      showOverlay('add-master-token', null, dismissOverlay)
+    } catch (e) {
```

Comment:
no need to catch the error if it's not going to be used, this is perfectly acceptable:

```ts
try {
  await getAllResources()
  showOverlay('add-master-token', null, dismissOverlay)
} catch {
  dispatch(notify(getResourcesTokensFailure('all access token')))
}